.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch",
+      "type": "go",
+      "request": "launch",
+      "mode": "auto",
+      "program": "${workspaceFolder}/cmd/hcloud/main.go",
+      "env": {},
+      "args": ["server", "create", "--image=1", "--type=1", "--name=testi"]
+    }
+  ]
+}
```

Comment:
I would prefer to not have editor-specific config files in the public repository.